tests/test_makeCoaddTempExp.py

```diff
@@ -96,28 +96,34 @@ def test_getCalibratedExposureGetCalexpRaises(self):
         self.assertIn('Exposure not found', str(cm.exception))
 
     def test_getCalibratedExposure(self):
-        task = MakeCoaddTempExpTask(config=self.config)
+        for includeCalibVar in [True, False]:
```

Comment:
I'm not keen on test for loops like this: they make it harder to debug failing tests. Especially when there are only two states, it's much nicer to take what you've made the internals of the loop and turn it into a method (`_checkCalibratedExposure`?), and then just call that twice with the two states.

Reply:
Agreed. How about 4 states? (Check out the latest unit test. This what you had in mind?)